modules/Bio/EnsEMBL/Mapper.pm

```diff
@@ -499,9 +499,9 @@ sub map_insert {
     # adjust coordinates, remove gaps
     my ($c1, $c2);
     if($strand == -1) {
-      ($c2,$c1) = @coords;
+      ($c2,$c1) = $include_original_region ? @coords->{'mapped'} : @coords;
```

Comment:
I don't think that syntax works in all versions of Perl; I can't get it to work in 5.14 or 5.26. Would be safer to use

`map {$_->{'mapped'}} @coords` to get the array slice here.

Reply:
Reworked.